tests/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['testserver', 'localhost',]
```

Comment:
@nemesisdesign What about this? Shouldn't this do to `local_settings.py`

Reply:
@atb00ker , my test will not pass if I don't add this. Because I'm using django.test.Client which uses a testserver. Since the build won't work without this, I added it to the main settings file.

Reply:
@atb00ker thanks for spotting this, it is indeed symptom of an issue

Reply:
once you address the previous issue you should not need this change anymore, although you can keep `localhost`